templates/patterns/banner/banner.html.twig

```diff
@@ -0,0 +1,46 @@
+{% spaceless %}
```

Comment:
Temporal until BCL 0.12.0

Reply:
move the contents to pattern-banner.html.twig and remove this file